ui-v2/app/helpers/dom-position.js

```diff
@@ -0,0 +1,29 @@
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+
+export default Helper.extend({
+  dom: service('dom'),
+  compute: function([selector], hash) {
+    const $el = this.dom.element(selector);
+    let $refs = [$el.offsetParent, $el];
+    // TODO: helper probably need to accept a `reference=` option
+    // with a selector to use as reference/root
+    if (selector.startsWith('#resolver:')) {
+      $refs.unshift($refs[0].offsetParent);
+    }
```

Comment:
Hehe yes definitely! I'm guessing this will be useful for something else down the line, so we'll probably add an optional reference dom element or selector to use for a 'root', and then move this to our `dom` service, hence the TODO. This works fine right now though.